server/controllers/userController.js

```diff
@@ -2,15 +2,18 @@
 /* https://mherman.org/blog/2015/02/12/postgresql-and-nodejs/ */
 import db from '../helpers/dbHelper';
 import bcrypt from '../helpers/bcrypt';
+import dotenv from 'dotenv';
 import jwt from 'jsonwebtoken';
 
+ dotenv.config()
+ const secret = process.env.SECRET_KEY;
+
 export default{
    createUser: (req, res, next) => {
             const {
                 userName, email, password, confirmPassword
             } = req.body;
             const sql = 'INSERT INTO users ("userName", email, password) values($1, $2, $3) RETURNING *';
```

Comment:
does the user entity have any unique column, if yes; can we ensure that no duplication is allowed